### PR TITLE
Update windows-cheatsheet.md

### PR DESCRIPTION
**Pull request by vinay4563**
added windows shortcut key

### CODE REVIEW

**Review by tushar-kumar — APPROVED** [review]
Approved

File: CheatSheets/windows-cheatsheet.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ created: 2022-10-20
 |` Win + Shift + S`| Take Screenshot of selected area|
 |` Win + Upper Arrow key`| Maximize windows|
 | `Win + number key` | to open application from taskbar with numbered key position
+| `Win + Alt + R` | to start in-built screen recording |
```
Comment (crescentpartha) [inline CheatSheets/windows-cheatsheet.md:83]:
```suggestion
| `Win + Alt + R` | Allows you to instantly start or stop recording your screen |
```

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Please, modify your code.

**Review by crescentpartha — APPROVED** [review]
Approved but have to modify the code by myself.

**Pull request merged**